doc/src/Howto_apip.rst

```diff
@@ -0,0 +1,200 @@
+Adaptive-precision interatomic potentials (APIP)
+================================================
+
+The :ref:`APIP package <PKG-APIP>` allows to use adaptive-precision potentials
+according to :ref:`(Immel) <Immel2025_1>`.
+The potential energy :math:`E_i` of an atom :math:`i` described by an
```

Comment:
I suggest adding a sentence or two here which explains/motivates up front what an APIP is (in words) before
showing the energy formula.  Something like:

An APIP model is one which combines a traditional empirical potential (fast, less accurate) with
a machine learned potential (slower, more accurate) using a lambda parameter to weight
the contribution of each component to an atom's energy and force.  As explained below, lambda
values can be defined for each atom and adjusted dynamically during a simulation.